app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -57,53 +51,91 @@ docker pull kong:{{page.kong_versions[page.version-index].ce-version}}-alpine
-<div class="copy-code-snippet"><pre><code>docker tag kong/kong-gateway:{{page.kong_versions[page.version-index].ee-version}}-alpine <div contenteditable="true">{TAG_NAME}</div></code></pre></div>
-{% endnavtab %}
-{% navtab Kong Gateway (OSS) %}
-<div class="copy-code-snippet"><pre><code>docker tag kong:{{page.kong_versions[page.version-index].ce-version}}-alpine <div contenteditable="true">{TAG_NAME}</div></code></pre></div>
-{% endnavtab %}
-{% endnavtabs %}
-{% endcapture %}
-{{ tag_image | indent | replace: " </code>", "</code>" }}
+## Install Kong Gateway with a database
+
+Set up a {{site.base_gateway}} container with a PostgreSQL database to store
+Kong configuration.
 
 ### Prepare the database
 
-1. Create a custom Docker network to allow the containers to discover and communicate with each other:
+1. Create a custom Docker network to allow the containers to discover and
+communicate with each other:
+
+    ```sh
+    docker network create kong-net
+    ```
 
-   <pre><code>docker network create <div contenteditable="true">{NETWORK_NAME}</div></code></pre>
+    You can name this network anything you want. We use `kong-net` as
+    an example throughout this guide.
 
 1. Start a PostgreSQL container:
 
-   <pre><code>docker run -d --name <div contenteditable="true">{PG_CONTAINER_NAME}</div> \
-     --network=<div contenteditable="true">{NETWORK_NAME}</div> \
+    ```sh
+    docker run -d --name kong-database \
+     --network=kong-net \
      -p 5432:5432 \
-     -e "POSTGRES_USER=<div contenteditable="true">{DATABASE_USER}</div>" \
-     -e "POSTGRES_DB=<div contenteditable="true">{DATABASE_NAME}</div>" \
-     -e "POSTGRES_PASSWORD=<div contenteditable="true">{DATABASE_PASSWORD}</div>" \
+     -e "POSTGRES_USER=kong" \
+     -e "POSTGRES_DB=kong" \
+     -e "POSTGRES_PASSWORD=kongpass" \
      postgres:9.6
-   </code></pre>
+    ```
+
+    * `POSTGRES_USER` and `POSTGRES_DB`: Set these values to `kong`. This is
+    the default value that Kong Gateway expects.
+    * `POSTGRES_PASSWORD`: Set the database password to any string.
+
+    In this example, the Postgres container named `kong-database` can
+    communicate with any containers on the `kong-net` network.
 
 1. Prepare the Kong database:
+
 {% capture migrations %}
-<pre><code>docker run --rm --network=<div contenteditable="true">{NETWORK_NAME}</div> \
+{% navtabs codeblock %}
+{% navtab Kong Gateway %}
+```sh
+docker run --rm --network=kong-net \
  -e "KONG_DATABASE=postgres" \
- -e "KONG_PG_HOST=<div contenteditable="true">{PG_CONTAINER_NAME}</div>" \
- -e "KONG_PG_PASSWORD=<div contenteditable="true">{CONTAINER_PASSWORD}</div>" \
- -e "KONG_PASSWORD=<div contenteditable="true">{PASSWORD}</div>" \
- <div contenteditable="true">{TAG_NAME}</div> <div contenteditable="true">{DATABASE_NAME}</div> migrations bootstrap</code></pre>
+ -e "KONG_PG_HOST=kong-database" \
+ -e "KONG_PG_PASSWORD=kongpass" \
+ -e "KONG_PASSWORD=test" \
+kong/kong-gateway kong migrations bootstrap
```

Comment:
Using `kong/kong-gateway` is the same as `kong/kong-gateway:latest` which may not match `docker pull kong/kong-gateway:{{page.kong_versions[page.version-index].ee-version}}-alpine` from above

Output:

```
docker run --rm --network=kong-net \
  -e "KONG_DATABASE=postgres" \
  -e "KONG_PG_HOST=kong-database" \
  -e "KONG_PG_PASSWORD=kongpass" \
  -e "KONG_PASSWORD=test" \
 kong/kong-gateway kong migrations bootstrap

Unable to find image 'kong/kong-gateway:latest' locally
latest: Pulling from kong/kong-gateway
Digest: sha256:8d69f2ad968154c4751410a5f4c3cfe51bc5c325c9e5a5f47d711afeb05d2c2d
```

Reply:
Hmm. I didn't realize. There's two ways to go about this, then: either bring back the step to tag the image that you pulled, or include the version here. 
- Tagging would make each codeblock cleaner, but adds another step
- Using `kong/kong-gateway:<version>` makes for a longer codeblock but removes the need to pull and tag the image initially.

Reply:
Use `kong/kong-gateway:version`. Retagging local images isn't a common workflow

Reply:
After looking at your other comments, looks like tag version makes things simpler. I'll go with that.

In this case: does it mean that they don't actually need to pull the image before going through the rest of the steps?

Reply:
Yep! The first time they use it the image will be pulled automatically